From 64afb125ce17483937f3988a7b76ef0e100c2fcc Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 12:21:21 +0100
Subject: [PATCH] Revert "man: ip-link: Remove bits about proxy_arp and
 proxy_arp_wifi"

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1374360
Upstream Status: RHEL only.

This reverts commit 75b314be0b22357ef564e451bc8d8dd1b9de74aa.
---
 man/man8/ip-link.8.in | 15 ++++++++++++++-
 1 file changed, 14 insertions(+), 1 deletion(-)

diff --git a/man/man8/ip-link.8.in b/man/man8/ip-link.8.in
index cfd996a..73cd86d 100644
--- a/man/man8/ip-link.8.in
+++ b/man/man8/ip-link.8.in
@@ -1130,6 +1130,10 @@ the following additional arguments are supported:
 ] [
 .BR flood " { " on " | " off " }"
 ] [
+.BR proxy_arp " { " on " | " off " }"
+] [
+.BR proxy_arp_wifi " { " on " | " off " }"
+] [
 .BI mcast_router " MULTICAST_ROUTER"
 ] [
 .BR mcast_fast_leave " { " on " | " off "} ]"
@@ -1170,7 +1174,16 @@ port to be reflected back.
 
 .BR flood " { " on " | " off " }"
 - open the flood gates on this port, i.e. forward all unicast frames to this
-port also.
+port also. Requires
+.BR proxy_arp " and " proxy_arp_wifi
+to be turned off.
+
+.BR proxy_arp " { " on " | " off " }"
+- enable proxy ARP on this port.
+
+.BR proxy_arp_wifi " { " on " | " off " }"
+- enable proxy ARP on this port which meets extended requirements by IEEE
+802.11 and Hotspot 2.0 specifications.
 
 .BI mcast_router " MULTICAST_ROUTER"
 - configure this port for having multicast routers attached. A port with a
-- 
1.8.3.1

